staging/src/k8s.io/csi-api/pkg/apis/csi/v1alpha1/types.go

```diff
@@ -18,6 +18,15 @@ package v1alpha1
 
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+type PluginMechanism string
+
+const (
+	// InTreePluginMechanism means components should use the in-tree plugin for volume operations
+	InTreePluginMechanism PluginMechanism = "in-tree"
```

Comment:
I believe we usually use a prefix, so `VolumePluginMechanismInTree` should be the name here (also next line).